src/haproxy.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+#
+# igcollect - haproxy
+#
+# Copyright (c) 2017 InnoGames GmbH
+#
+
+from argparse import ArgumentParser
+from subprocess import Popen, PIPE
+from time import time
+
+
+def parse_args():
+    parser = ArgumentParser()
+    parser.add_argument('--prefix', default='haproxy')
+    return parser.parse_args()
+
+
+def main():
+    args = parse_args()
+    haproxy_info = Popen(
+        ('echo "show stat" | socat /var/run/haproxy.stat stdio'),
+        shell=True,
+        stdout=PIPE).stdout.read()
+    haproxy_info = haproxy_info.splitlines()
+    header = haproxy_info[0].replace(" ", "_").split(',')
+    data_overall = []
+    for n in xrange(1, len(haproxy_info) - 1):
```

Comment:
`xrange()` is no more on Python 3.  We have Python 3 only servers.  Please test your code on Python 3.

Reply:
Better use `enumerate()` there